src/development/tools/devtools/memory.md

```diff
@@ -12,58 +12,74 @@ managed by the Dart VM (virtual machine).
 
 ## DevTools memory page
 
-DevTools Memory page lets you peek at how an isolate is using
+The DevTools Memory page lets you peek at how an isolate is using
 memory at a given moment. 
 
 Memory profiling in DevTools consists of 3 main functions:
+
 * Charting memory usage statistics and events
-* Anaylsis to view all memory via a heap to detect memory issues and inspect objects
-* Allocations to monitor and track (stack trace) specific classes and objects when an allocation occurs
+* Anaylsis to view all memory via a heap to detect memory issues
+  and inspect objects
+* Allocations to monitor and track (stack trace) specific classes
+  and objects when an allocation occurs
 
 ## Charting memory statistics and events
 
-At the top-level, when the memory tab is selected memory statistics from
-the VM are collected. These statistics are displayed in the two overview
-charts (Dart memory and Android-only) the collection of general memory
-usage e.g., total heap used, external heap, maximum heap capacity,
-Resident Set Size (RSS). As you interact with your application various
-events are detected e.g., memory GC (garabage collection),Flutter events,
-user fired events (using the ```dart:developer``` package) are collected
-in the same timeline as the memory statistics. All of this collected
-statistics and events are displayed in charts see [Memory anatomy](#memory-anatomy).
+At the top-level, when the **Memory** tab is selected,
+memory statistics from the VM are collected and displayed
+in the two overview charts (Dart memory and Android only).
+The collected statistics include general memory usage, such as
+total heap used, external heap size, maximum heap capacity, and
+Resident Set Size (RSS). As you interact with your application,
+various events are detected and collected in the same timeline
+as the memory statistics, such as memory GC (garabage collection),
+Flutter events, and user fired events (using the `dart:developer` package).
+All of these collected statistics and events are displayed in charts.
+For more information, see [Memory anatomy](#memory-anatomy).
 
 ## Analysis and snapshots
 
-A Snapshot is a complete, the most complex and time consuming view of
-all objects in the Dart memory heap. Each time a snapshot is taken, an
-analysis is performed over the collected memory data.  The analysis
-attempts to identify any memory patterns that may cause leaks or lead to
-application crashes. For example, loading large assets for thumbnail-sized
-images inefficiently, memory usage can be improved by loading smaller
-assets or adjusting the cacheWidth/cacheHeight to decode an image to a
-smaller size reducing the memory usage of the ImageCache. The analysis
-catches issues like this see [Analysis tab](#analysis-tab).
+A `Snapshot` is a complete view of
```

Comment:
Snapshot doesn't represent a defined class here, I guess "snapshot" would be fine.

Reply:
Done